### PR TITLE
Add Graph/Info 

### PR DESCRIPTION
**Pull request by gmorell**
Added support for:
- graphing
- returning information about existing rrd databases

The changes for some usages of `std::os::raw` to `core::ffi` are for compatibility with librrd / our target

TODO: I still need to put together some examples for using info and graph

Should be squashed-on-merge :)

### CODE REVIEW

File: src/sys.rs [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
+    pub ptr: *mut c_uchar,
+}
+
+#[repr(C)]
+pub union rrd_infoval_t {
+    pub u_cnt: c_ulong,
+    pub u_val: rrd_value_t,
+    pub u_str: *mut c_char,
+    pub u_int: c_int,
+    pub u_blo: std::mem::ManuallyDrop<rrd_blob_t>,
```
Comment (cygnus9) [inline src/sys.rs:30]: Why is there a `ManuallyDrop` here?

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rrd"
-version = "0.1.1"
+version = "0.1.6"
```
Comment (cygnus9) [inline Cargo.toml:3]: Can you make this `"0.1.2"`? I don't think there's any need to skip 4 patches.

**Comment by cygnus9** [comment]
Hi,

Thanks for spending your time on this. I just have a minor comment and a question. I'll be happy to squasah and merge otherwise.

**Comment by gmorell** [comment]
I've updated my branch to match the structural changes from upstream, confirm that it info and graphing work with the client application still.

**Comment by cygnus9** [comment]
Thank you

**Pull request merged**